docs_src/parameter_types/enum/tutorial006_an.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum
+from typing import List
+
+import typer
+from typing_extensions import Annotated
+
+
+class Food(str, Enum):
+    f1 = "Eggs"
+    f2 = "Bacon"
+    f3 = "Cheese"
+
+
+def main(
+    groceries: Annotated[List[Food], typer.Option(enum_by_name=True)] = ["f1", "f3"],
+):
+    print(f"Buying groceries: {', '.join([f.value for f in groceries])}")
+
```

Comment:
The linter is unhappy with this code, but does accept similar code in `tutorial003_an.py`:
```
def main(groceries: Annotated[List[Food], typer.Option()] = [Food.food_1, Food.food_3]):
```
🤔

Reply:
Fixed!